tests/update_handlers/ca_update_handler_test.py

```diff
@@ -19,6 +20,10 @@
 from tests.test_helpers.ca_fakes import FakeContext
 
 
+def epics_timestamp():
+    return timestamp_to_epics(time.time())
```

Comment:
Instead of `time.time`, we can use `time.perf_counter` which is more accurate.

Reply:
`perf_counter()` does not work here.